web/src/components/njwds/Hero.tsx

```diff
@@ -57,24 +52,13 @@ export const Hero = (): ReactElement => {
               <div className="text-base-darkest font-sans-lg margin-bottom-3 desktop:margin-bottom-3">
                 {landingPageConfig.heroSupportingText}
               </div>
-              <div className="desktop:display-inline margin-bottom-2 desktop-margin-bottom-0 desktop:margin-right-2">
-                <Button
-                  style="primary-big"
-                  onClick={primaryCTAOnClick}
-                  dataTestid="hero-login-button"
-                  widthAutoOnMobile
-                  noRightMargin
-                >
-                  {landingPageConfig.heroCallToActionText}
-                </Button>
-              </div>
-
-              <AuthButton position="HERO" landing />
```

Comment:
Same thing here regarding code cleanup. If we are removing AuthButton from the hero component, then the AuthButton component doesn't need to support "HERO" as a prop position anymore, since it's not getting used anywhere else.

These kinds of changes are often known as the "scout rule" aka "leaving the campground cleaner than when you found it." Even though your feature does not require changes to the AuthButton component, your change allows the AuthButton component to be changed to have a good deal less complexity, which is always desirable in the codebase. We don't want to support props like "HERO" that we don't use anymore. So part of being a good steward of the codebase is to notice the effect that your changes has on other components you might not otherwise have changed.

This technique doesn't require you to know every single component and what it does, but just be observant of what's being changed. I didn't previously know/remember that AuthButton took a position prop. But seeing this line of code being removed, with a prop labeled "HERO", I'm thinking "huh, if we're removing this from the Hero component, I bet that component doesn't need to take a prop called 'HERO' anymore". Then you can go to the component, check your assumptions, and clean up the code accordingly.